CONTRIBUTING.md

```diff
@@ -0,0 +1,78 @@
+# Contributing to the IDS testbed
+
+This is the official repository of the [IDSA](https://www.internationaldataspaces.org) testbed.
+
+All content published here is approved by the IDSA Technical Steering Committee and the IDSA Working Groups. Detailed information on the contribution process can be found in the [IDS-G Handbook](Handbook/README.md). Nevertheless, you are very welcome to contribute
+to this project when you find a bug, want to suggest an improvement, or have an idea for a useful
+feature. For this, always create an issue and a corresponding branch, and follow our style
+guides as described below.
+
+Please note that we have a [code of conduct](./CODE_OF_CONDUCT.md) that all contributors should stick to.
+
+## Changelog
+
+We document changes in the [CHANGELOG.md](./CHANGELOG.md) on root level which is formatted and
+maintained according to the rules documented on http://keepachangelog.com.
+
+## Issues
+
+You always have to create an issue if you want to propose or integrate a bugfix, improvement, or feature.
+Briefly and clearly describe the purpose of your contribution in the corresponding issue.
+The pre-defined [labels](#labels) improve the understanding of your intentions and help to follow
+the scope of your changes.
+
+**Bug Report**: As mentioned above, bug reports should be submitted as an issue. To give others
+the chance to reproduce the error in order to find a solution as quickly as possible, the report
+should at least include the following information:
+* Description: What did you expect and what happened instead?
+* Steps to reproduce (system specs included)
+* Relevant logs and/or media (optional): e.g. an image
+
+## Labels
+
+The [labels](https://github.com/International-Data-Spaces-Association/IDS-testbed/labels) are listed at the
+[issues](https://github.com/International-Data-Spaces-Association/IDS-testbed/issues).
+There are two types of labels: one describes the content of the issue and should be used by the
+developer that creates the issue. The other one, starting with `status`, will be added from the
+developer that takes on the issue. New issues should be initially marked with `status:open`.
+*  Basic labels: `bug`, `enhancement`, `suggestion`, `documentation` `outdated`, `question`, `discussion`
+*  `status:closed`: issue is closed (after successful approval by issuer and QA)
+*  `status:duplicate`: issue is a duplicate of another linked issue and therefore discontinued
+*  `status:in-progress`: issue has been assigned and is currently being worked on
+*  `status:on-hold`: issue may be implemented at a later date
+*  `status:open`: issue has been submitted or re-opened recently
+*  `status:out-of-scope`: issue is considered out of the project's scope and therefore not further considered
+*  `status:resolved`: issue has been implemented and tested by a developer
+*  `status:wont-fix`: issue is in scope but considered impossible or too expensive to deal with
```

Comment:
The labels described here doesn't match the actual labels:
https://github.com/International-Data-Spaces-Association/IDS-testbed/labels